tests/Application/templates/bundles/SyliusShopBundle/Homepage/_newsletter.html.twig

```diff
@@ -0,0 +1,18 @@
+<div class="ui very padded secondary segment newsletter">
+    <div class="ui bottom aligned grid">
+        <div class="doubling two column row">
+            <div class="column">
+                <h2 class="ui huge header">
+                    {{ 'sylius.homepage.newsletter'|trans }}
+                </h2>
+                <p>
+                    {{ 'sylius.homepage.newsletter_description'|trans|raw }}
+                </p>
+            </div>
+            <div class="column">
+                {% include '@BitBagSyliusMailChimpPlugin/_subscribe.html.twig' %}
+            </div>
+        </div>
+    </div>
+</div>
+<div class="ui hidden divider"></div>
```

Comment:
In `src/Resources/views/_subscribe.html.twig` file You've added empty space between block element siblings, consider keeping it consistent